### DISCUSSION TITLE
Comparing a multi relation to another multi relation in API rules

### DISCUSSION THREAD

**Discussion by EverlyScott** [pocketbase/pocketbase#4665] [Q&A] [answered]
Hi, I am building an app with react and I only want users who are in similar groups to be able to view each other's profiles. My `users` table contains a `joinedGroups` multiple select field, and I'm trying to compare the two in API rules. I currently have this:
```
id = @request.auth.id || joinedGroups.id ?= @request.auth.joinedGroups.id
```
So I check if the accessed user's groups list contains one of the id's from the authed user's groups, but this doesn't seem to work.

Any insight on how to properly do this?

**Comment by ganigeorgiev** [comment, marked as answer]
Based on your description the above rule should work. Make sure that you are sending the request with the correct user.

If you are not able to resolve it, please export your collections configuration as json (_Admin UI > Settings > Export collections_) to attempt to reproduce it locally.

**Reply by EverlyScott** [reply]
i forgot to add the group to my test user's `joinedGroups` list 🤦
